apps/condo/domains/marketplace/components/Invoice/MarketplaceInvoicesContent.tsx

```diff
@@ -0,0 +1,181 @@
+import { DEFAULT_PAGE_SIZE, Table } from '@condo/domains/common/components/Table/Index'
+import { TableFiltersContainer } from '@condo/domains/common/components/TableFiltersContainer'
+import { useQueryMappers } from '@condo/domains/common/hooks/useQueryMappers'
+import { useSearch } from '@condo/domains/common/hooks/useSearch'
+import { getFiltersQueryData } from '@condo/domains/common/utils/filters.utils'
+import { updateQuery } from '@condo/domains/common/utils/helpers'
+import { getFiltersFromQuery, getPageIndexFromOffset, parseQuery } from '@condo/domains/common/utils/tables.utils'
+import { useMarketplaceInvoicesFilters } from '@condo/domains/marketplace/hooks/useMarketplaceInvoicesFilters'
+import { useMarketplaceInvoicesTableColumns } from '@condo/domains/marketplace/hooks/useMarketplaceInvoicesTableColumns'
+import { Invoice } from '@condo/domains/marketplace/utils/clientSchema'
+
+
+const TableContent = () => {
+    const intl = useIntl()
+    const SearchPlaceholder = intl.formatMessage({ id: 'filters.FullSearch' })
+    const CreateInvoiceMessage = intl.formatMessage({ id: 'pages.condo.marketplace.invoice.invoiceList.createInvoice' })
+
+    const router = useRouter()
+    const { organization } = useOrganization()
+    const { filters, offset, sorters } = parseQuery(router.query)
+    const currentPageIndex = getPageIndexFromOffset(offset, DEFAULT_PAGE_SIZE)
+    const filtersMeta = useMarketplaceInvoicesFilters()
+    const columns = useMarketplaceInvoicesTableColumns({ filtersMeta })
+    const { filtersToWhere, sortersToSortBy } = useQueryMappers(filtersMeta, ['createdAt', 'number', 'toPay'])
+    const sortBy = useMemo(() => sortersToSortBy(sorters) as SortInvoicesBy[], [sorters, sortersToSortBy])
+
+    const handleRowAction = useCallback((record) => {
+        return {
+            onClick: async () => {
+                await router.push(`/marketplace/invoice/${record.id}`)
+            },
+        }
+    }, [router])
+
+    const {
+        loading: invoicesLoading,
+        count: totalInvoices,
+        objs: invoices,
+    } = Invoice.useObjects({
+        sortBy,
+        where: {
+            ...filtersToWhere(filters),
+            context: {
+                organization: { id: organization.id },
+            },
+        },
+        first: DEFAULT_PAGE_SIZE,
+        skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
+    })
+
+    const [search, handleSearchChange] = useSearch()
+    const handleSearch = useCallback((e) => handleSearchChange(e.target.value), [handleSearchChange])
+
+    const filtersFromQuery = useMemo(() => getFiltersFromQuery(router.query), [router.query])
+    const createdAtValueFromQuery = useMemo(() => get(filtersFromQuery, 'createdAt'), [filtersFromQuery])
+    const initialValue = createdAtValueFromQuery ?
+        [dayjs(createdAtValueFromQuery[0]), dayjs(createdAtValueFromQuery[1])] :
+        [dayjs().subtract(6, 'days'), dayjs()]
```

Comment:
This const is only used in the hook
Maybe move it there? Otherwise it’s not clear what the initial data is